client/components/auth/ResetPassword.vue

```diff
@@ -7,64 +7,66 @@
       class="mb-5">
       {{ error }}
     </v-alert>
-    <form @submit.prevent="submit">
-      <v-text-field
-        ref="password"
-        v-model="password"
-        v-validate="{ required: true, min: 6, alphanumerical: true }"
-        :error-messages="vErrors.collect('password')"
-        prepend-inner-icon="mdi-lock"
-        type="password"
-        name="password"
-        label="Password"
-        placeholder="Password"
-        outlined
-        class="mb-1" />
-      <v-text-field
-        v-model="passwordConfirmation"
-        v-validate="{ required: true, confirmed: 'password' }"
-        :error-messages="vErrors.collect('passwordConfirmation')"
-        type="password"
-        name="passwordConfirmation"
-        label="Re-enter password"
-        placeholder="Password confirmation"
-        data-vv-as="password"
-        prepend-inner-icon="mdi-lock-outline"
-        outlined />
-      <v-btn
-        :disabled="!isValid"
-        type="submit"
-        color="primary darken-1"
-        class="my-1">
-        Change password
-      </v-btn>
-    </form>
+    <validation-observer v-slot="{ invalid }" ref="form">
+      <form @submit.prevent="submit">
```

Comment:
You don't need this `ref` if you use `handleSubmit` scoped prop.

```suggestion
    <validation-observer v-slot="{ invalid, handleSubmit }">
      <form @submit.prevent="handleSubmit(submit)">
```